tools/cldr-code/src/main/java/org/unicode/cldr/test/ExampleGenerator.java

```diff
@@ -2219,22 +2294,33 @@ private String addTransliteration(String input, String value) {
      * @return string with HTML for the background.
      */
     private String finalizeBackground(String input) {
```

Comment:
This function shows why the example content should be shipped as *structured data* to the front end.  This is not the best practice for generating HTML.

Reply:
Let's carve that off into a separate ticket. I think the product of the Example generator could be changed from a string to a class. The data in the class can be is convenient to ship to the client, but can also be convenient to convert to a purely textual form that we can use in the low-level tests/tools.